src/wallet/wallet.cpp

```diff
@@ -251,11 +259,18 @@ bool CWallet::AddWatchOnly(const CScript &dest)
 {
```

Comment:
The second two calls to AddWatchOnly in processImport probably could take the timestamp argument.